src/main/java/heronarts/lx/midi/surface/MidiFighter64.java

```diff
@@ -0,0 +1,203 @@
+    40, 41, 42, 43,  72, 73, 74, 75,
+    44, 45, 46, 47,  76, 77, 78, 79,
+    48, 49, 50, 51,  80, 81, 82, 83,
+    52, 53, 54, 55,  84, 85, 86, 87,
+    56, 57, 58, 59,  88, 89, 90, 91,
+    60, 61, 62, 63,  92, 93, 94, 95,
+    64, 65, 66, 67,  96, 97, 98, 99
+  };
+
+  public static final String DEVICE_NAME = "Midi Fighter 64";
+
+  public static final int LED_OFF = 0;
+  public static final int LED_GRAY_DIM = 1;
+  public static final int LED_GRAY = 2;
+  public static final int LED_WHITE = 3;
+  public static final int LED_PINK = 4;
+  public static final int LED_RED = 5; // And 60
+  public static final int LED_RED_HALF = 6;
+  public static final int LED_RED_DIM = 7;
+  public static final int LED_WARM_WHITE = 8;
+  public static final int LED_ORANGE = 9; // And 61
+  public static final int LED_ORANGE_HALF = 10;
+  public static final int LED_ORANGE_DIM = 11;
+  public static final int LED_PALE_YELLOW = 12;
+  public static final int LED_YELLOW = 13;
+  public static final int LED_YELLOW_HALF = 14;
+  public static final int LED_YELLOW_DIM = 15;
+  public static final int LED_PALE_GREEN_BLUE = 16;
+  public static final int LED_GREEN_HALF = 17; // And 18, 21, 22, 25, 26
+  public static final int LED_GREEN = 19; /// And 23, 27
+  public static final int LED_PALE_BLUE_GREEN = 20; // And 24
+  public static final int LED_AQUA = 28;
+  public static final int LED_AQUA_HALF = 29; // And 30
+  public static final int LED_AQUA_DIM = 31;
+  public static final int LED_BLUE_AQUA = 32;
+  public static final int LED_BLUE_AQUA2 = 33; // If you have a better name, send a PR
+  public static final int LED_BLUE_AQUA_HALF = 34;
+  public static final int LED_BLUE_AQUA_DIM = 35;
+  public static final int LED_SKY = 36;
+  public static final int LED_AZURE = 37;
+  public static final int LED_AZURE_HALF = 38;
+  public static final int LED_AZURE_DIM = 39;
+  public static final int LED_PERIWINKLE = 40;
+  // 41-47 are more shades of blue than I can name
+  public static final int LED_LAVENDER = 48;
+  public static final int LED_PURPLE = 49;
+  public static final int LED_PURPLE_HALF = 50;
+  public static final int LED_PURPLE_DIM = 51;
+  public static final int LED_PALE_MAGENTA = 52;
+  public static final int LED_MAGENTA = 53;
+  public static final int LED_MAGENTA_HALF = 54;
+  public static final int LED_MAGENTA_DIM = 55;
+  public static final int LED_MAGENTA_PINK = 56;
+  public static final int LED_HOT_PINK = 57;
+  public static final int LED_HOT_PINK_HALF = 58;
+  public static final int LED_HOT_PINK_DIM = 59;
+  public static final int LED_GOLDENROD = 62;
+  public static final int LED_LAWN_GREEN = 63;
+  // There are 64 more colors, but they seem like dupes.
+  // There might be a small number of slightly unique ones in that range.
+
+  // Holds the colors that will be sent out to the device. This array
+  // starts with (0,0), (0,1), ... (7,7) of the LEFT page, and then
+  // the same ordering for the RIGHT page.
+  private int[] colors;
```

Comment:
I would make this `private final int[]` since you always copy into this array, you don't replace it. Some constants for the size checks would be great too.

```
public static final int NUM_ROWS = 8;
public static final int NUM_COLUMNS = 8;
public static final int GRID_SIZE = 64;
public static final int NUM_PAGES = 2;
public static final int NUM_COLORS = GRID_SIZE * NUM_PAGES;
```

There's a lot of magic 64/128 floating around.